hikari/impl/event_manager_base.py

```diff
@@ -371,31 +371,36 @@ def unsubscribe(
     def listen(
         self,
         event_type: typing.Optional[typing.Type[event_manager_.EventT_co]] = None,
```

Comment:
you can really just do
```py
def listen(self, *event_types: typing.Type[event_manager_.EventT_co])
```

then implicitly check if the tuple is empty with an if statement since we shouldn't care about the first arg's name here